api/eds.proto

```diff
@@ -72,8 +74,8 @@ message LbEndpoint {
   // is "canary", providing the optional canary status of the upstream host.
   // This may be matched against in a route's ForwardAction metadata_match field
   // to subset the endpoints considered in cluster load balancing.
-  // TODO(htuch: [V2-API-DIFF] Need to plumb this through Envoy and have
-  // everywhere that canary is used be capable of working on metadata.
+  // [#v2-api-diff: TODO(htuch: [V2-API-DIFF] Need to plumb this through Envoy and have
```

Comment:
please remove TODOs from docs or put them into a doc comment (comment in comment)

Reply:
I removed as many as possible. For this one I wasn't sure if it should go in a comment tag or a  v2 diff tag. @htuch which one do you want for this?

Reply:
Best to make this call on a case-by-case basis. I'd nuke this comment.

Reply:
looks terrible as it is. Will change into doc comment like you suggest.